deploy-prod.sh

```diff
@@ -3,14 +3,15 @@
 # Copy or symlink it to some deployment directory.
 
 OPENFISCA_PORT=12000
+TARGET=${1:-origin/master}
 
 set -ex
 
 cd /var/www/dds
 
 # Update mes-aides
 git fetch origin
-git checkout --force --detach origin/master
+git checkout --force --detach $TARGET
```

Comment:
Would be nice for branch, and definitely appropriate in order to align production and staging deployments, but would prevent reverting to a commit through its SHA, which is the main use case here.